<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="KAPPO-NOTE" ignored="0" name="Surefire test" tests="3" failures="2" timestamp="2021-02-16T10:59:03 BRT" time="3.672" errors="0">
  <testcase name="runScenario" time="2.36" classname="io.cucumber.testng.AbstractTestNGCucumberTests">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at steps.StepDefinitions.validarCadastroDeNovoCliente(StepDefinitions.java:160)
at ✽.Eu devo ver que o corpo de resposta retornado esta dentro do especificado(file:///home/mkapobianco/Projects/TestesAutomatizadosBrasilPrev/src/test/java/features/BuscarClientes.feature:8)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase name="runScenario" time="0.192" classname="io.cucumber.testng.AbstractTestNGCucumberTests">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;200&amp;gt; but was:&amp;lt;404&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<200> but was:<404>
at org.junit.Assert.fail(Assert.java:89)
at org.junit.Assert.failNotEquals(Assert.java:835)
at org.junit.Assert.assertEquals(Assert.java:647)
at org.junit.Assert.assertEquals(Assert.java:633)
at steps.StepDefinitions.validarStatusCode(StepDefinitions.java:155)
at ✽.Eu devo verificar que o HTTP StatusCode retornado é 200(file:///home/mkapobianco/Projects/TestesAutomatizadosBrasilPrev/src/test/java/features/BuscarClientes.feature:14)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase name="runScenario" time="0.432" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
</testsuite> <!-- Surefire test -->
